API docs landing page (#1679)
pulumi/docs: content/docs/reference/pkg/_index.md

infrastructure.

> These APIs are available as packages in your chosen language's package manager

Reviewer: ```suggestion
These APIs are available as packages in your chosen language's package manager
```

> &mdash; npm for TypeScript/JavaScript and PyPI for Python. There is a dedicated package for

Reviewer: ```suggestion
---npm for TypeScript/JavaScript and PyPI for Python. There is a dedicated package for
```

each cloud that includes access to its full capabilities. In addition, Pulumi
offers many convenience packages that make common tasks easier, like setting
up a network, creating a Kubernetes cluster, and building and publishing containers
to private registries.

### Choose Your Language

{{< langchoose nogo >}}

### General Purpose Packages

> The Pulumi SDK package is used for accessing the core programming model around resources, configuration, etc.

Reviewer: I don't like using etc. in technical documentation, so how about "configuration, and other components"?

> The Pulumi SDK package is used for accessing the core programming model around resources, configuration, etc.

Author: ```suggestion
The Pulumi SDK package is used for accessing the core programming model around resources, configuration, and other components
```

* [**Cloudflare** (`pulumi_cloudflare`)]({{< relref "python/pulumi_cloudflare" >}})
* [**Datadog** (`pulumi_datadog`)]({{< relref "python/pulumi_datadog" >}})
* [**DigitalOcean** (`pulumi_digitalocean`)]({{< relref "python/pulumi_digitalocean" >}})
* [**DNSimple** (`pulumi_dnsimple`)]({{< relref "python/pulumi_dnsimple" >}})
* [**F5 BigIP** (`pulumi_f5bigip`)]({{< relref "python/pulumi_f5bigip" >}})
* [**GitLab** (`pulumi_gitlab`)]({{< relref "python/pulumi_gitlab" >}})
* [**Linode** (`pulumi_linode`)]({{< relref "python/pulumi_linode" >}})
* [**MySQL** (`pulumi_mysql`)]({{< relref "python/pulumi_mysql" >}})
* [**New Relic** (`pulumi_newrelic`)]({{< relref "python/pulumi_newrelic" >}})
* [**OpenStack** (`pulumi_openstack`)]({{< relref "python/pulumi_openstack" >}})
* [**Packet** (`pulumi_packet`)]({{< relref "python/pulumi_packet" >}})
* [**PostgreSQL** (`pulumi_postgresql`)]({{< relref "python/pulumi_postgresql" >}})
* [**TLS** (`pulumi_tls`)]({{< relref "python/pulumi_tls" >}})
* [**vSphere** (`pulumi_vsphere`)]({{< relref "python/pulumi_vsphere" >}})
{{% /lang %}}

### Cloud-Agnostic Packages

{{% lang nodejs %}}

> Pulumi offers a highly productive, cloud-agnostic package for container and serverless oriented programming in the

Reviewer: ```suggestion
Pulumi offers a highly productive, cloud-agnostic package for container and serverless programming in the
```
